data/authors.toml

```diff
@@ -15,5 +15,5 @@ home_link = "https://www.linkedin.com/in/rafalmaciag/"
 
 [sbortz]
 display_name = "Sebastian Bortz"
-avatar = "https://pbs.twimg.com/profile_images/1196904725951795205/i9o8mYX0_400x400.jpg"
+avatar = "https://pbs.twimg.com/profile_images/1509963876229689357/ZUmVWQ79_400x400.jpg"
```

Comment:
Do you mind downloading the image and adding it to https://github.com/event-modeling/eventmodeling.org/tree/master/static/avatars and using that instead?

Like https://github.com/event-modeling/eventmodeling.org/pull/26/files#diff-fb2c6dfdb52e5fe994810cb546767902efe4ef6e5765605df73eb9e90e462f4aR3

Reply:
Yep, good idea. 👍